Packs/Armorblox/Playbooks/Armorblox_Needs_Review.yml

```diff
@@ -203,31 +175,130 @@ tasks:
     task:
       brand: ""
       description: Prints text to war room (Markdown supported)
-      id: 3212c583-2a76-42c3-8cef-447ec14da87f
+      id: 936feb4c-4726-472c-87e3-4e6bc341d1d8
       iscommand: false
       name: Done
       script: Print
       type: regular
       version: -1
-    taskid: 3212c583-2a76-42c3-8cef-447ec14da87f
+    taskid: 936feb4c-4726-472c-87e3-4e6bc341d1d8
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": 530
+        }
+      }
+  "5":
+    id: "5"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      value:
+        simple: Mail Sent successfully to ${inputs.recipient_mail_address}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Prints text to war room (Markdown supported)
+      id: a1111d6d-6e09-4a04-89cd-65a1030040c4
+      iscommand: false
+      name: Mail Sent Successful
+      script: Print
+      type: regular
+      version: -1
+    taskid: a1111d6d-6e09-4a04-89cd-65a1030040c4
     timertriggers: []
     type: regular
     view: |-
       {
         "position": {
-          "x": -270,
+          "x": 710,
+          "y": 890
+        }
+      }
+  "6":
+    id: "6"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      "no":
+      - "7"
+      "yes":
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      brandname:
+        simple: Mail Sender (New)
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Returns 'yes' if integration brand is available. Otherwise returns
+        'no'
+      id: 2f97dd0b-f541-471d-87a8-3a6951c1e506
+      iscommand: false
+      name: Is Mail Sender Integration Enabled?
+      script: IsIntegrationAvailable
+      type: condition
+      version: -1
+    taskid: 2f97dd0b-f541-471d-87a8-3a6951c1e506
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 420,
           "y": 530
         }
       }
+  "7":
+    id: "7"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      value:
+        simple: Please configure Mail Sender (New) Integration to enable this feature
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Prints text to war room (Markdown supported)
```

Comment:
I see 2 instances of this text. Is this relevant to us?

Reply:
Are you talking about line 276?